image_classification/tensorflow/official/resnet/resnet_run_loop.py

```diff
@@ -278,19 +301,20 @@ def exclude_batch_norm(name):
   else:
     train_op = None
 
-  if not tf.contrib.distribute.has_distribution_strategy():
-    accuracy = tf.metrics.accuracy(
-        tf.argmax(labels, axis=1), predictions['classes'])
-  else:
-    # Metrics are currently not compatible with distribution strategies during
-    # training. This does not affect the overall performance of the model.
-    accuracy = (tf.no_op(), tf.constant(0))
+  accuracy = tf.metrics.accuracy(labels, predictions['classes'])
```

Comment:
Dropping the 'if not tf.contrib.distribute.has_distribution_strategy():' conditional results in runtime errors about trying to tf.assign to a mirrored variable. So it seems we may need to keep the if-else wrapping both accuracy metrics.

Reply:
Just to mirror my email response, this is caused by a tensorflow version which is out of date. It is definitely fixed by 1.11, and I'm not sure about 1.10